### PR TITLE
Edit Page Additions 

### PR DESCRIPTION
**Pull request by thedomcab**
https://github.com/hmcc-global/hmccaa-web/issues/173
image

image

https://github.com/hmcc-global/hmccaa-web/issues/171
image

image

https://github.com/hmcc-global/hmccaa-web/issues/175
image

image

https://github.com/hmcc-global/hmccaa-web/issues/179
image

image

https://github.com/hmcc-global/hmccaa-web/issues/178
image

image

https://github.com/hmcc-global/hmccaa-web/issues/180
image

https://github.com/hmcc-global/hmccaa-web/issues/181
image

### CODE REVIEW

File: frontend/src/components/page-connect/common-questions.js
Comment (terrianx) [inline frontend/src/components/page-connect/common-questions.js:1]: Screenshot 2024-03-18 at 3 56 48 PM
using chrome, i dont think this section is centered. it is on the figma

Comment (thedomcab) [reply frontend/src/components/page-connect/common-questions.js:1]: will look at later

File: frontend/src/components/page-main/getinvolved.js
Comment (terrianx) [inline frontend/src/components/page-main/getinvolved.js:1]: life group and member links are not updated.

File: frontend/src/pages/next-steps/membership.js
Comment (terrianx) [inline frontend/src/pages/next-steps/membership.js:1]: become a member and read more on distinctives links are not updated

File: frontend/src/pages/next-steps/baptism.js
Comment (terrianx) [inline frontend/src/pages/next-steps/baptism.js:1]: sign up for class link not updated. 

at certain screen width image becomes unreasonably small
Screenshot 2024-03-25 at 4 20 25 PM

File: frontend/src/pages/next-steps/ministryteams.js [hunk 1]
```diff
@@ -2,17 +2,266 @@ import * as React from "react";
...
+                src="../../images/ministry-teams-page.jpeg"
+              />
+            </div>
+            <div className="content-container flex flex-col gap-y-5 md:gap-y-10">
+              <div className="text-base md:text-lg">
+                <p className="mb-0">
+                  We believe God has gifted each and every single of us with
+                  unique giftings to build up the church and the world around
+                  us. We hope you are able to get involved and discover more
+                  about yourself and how you can be used for God&paos;s Kingdom.
```
Comment (terrianx) [inline frontend/src/pages/next-steps/ministryteams.js:147]: typo with apostrophe unicode

Screenshot 2024-03-25 at 4 27 13 PM

File: frontend/src/pages/next-steps/ministryteams.js [hunk 2]
```diff
@@ -2,17 +2,266 @@ import * as React from "react";
...
+              )}
+            </div>
+          </div>
+          <div className="flex flex-col max-w-[68.75rem] w-full items-center text-center gap-y-5 md:gap-y-14">
+            <PrimaryButtonLink to="/" hasArrow={true}>
+              Join A Team
+            </PrimaryButtonLink>
+            <div className="flex flex-col gap-y-3 md:gap-y-0">
+              <p className="mb-0 text-base md:text-lg">
+                *Note that you must be an active member opf HMCC to serve.
```
Comment (terrianx) [inline frontend/src/pages/next-steps/ministryteams.js:250]: extra 'p' before the f

File: frontend/src/pages/next-steps/transformation.js [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+                </p>
+                <p className="mb-0">
+                  Transformation classes are offered through the year to help
+                  challenge, encourage, and equip our members to know God more
+                  deeply and live out His calling for their lives.
+                </p>
+              </div>
+              <div className="flex">
+                <PrimaryButtonLink to="/" hasArrow={true}>
+                  Sign Up Now
```
Comment (terrianx) [inline frontend/src/pages/next-steps/transformation.js:112]: no sign up link

**Review by terrianx — CHANGES_REQUESTED** [review]
small fixes with some links and image sizes (not sure if image problems are that important, purely visual preference). also a couple of typos

File: frontend/src/pages/next-steps/membership.js
Comment (thedomcab) [reply frontend/src/pages/next-steps/membership.js:1]: where should become a member link to?

File: frontend/src/pages/next-steps/baptism.js
Comment (thedomcab) [reply frontend/src/pages/next-steps/baptism.js:1]: same here where should baptism link to?

File: frontend/src/pages/next-steps/membership.js
Comment (terrianx) [reply frontend/src/pages/next-steps/membership.js:1]: not sure actually, we can wait for maples response in the teams chat

**Comment by terrianx** [comment]
dom its pretty much ready to merged in, just waiting on maple to get back on the links

**Review by terrianx — APPROVED** [review]

**Pull request merged**